libs/partners/cohere/langchain_cohere/embeddings.py

```diff
@@ -45,6 +45,9 @@ class CohereEmbeddings(BaseModel, Embeddings):
     user_agent: str = "langchain"
     """Identifier for the application making the request."""
 
+    cohere_api_url: Optional[str] = None
```

Comment:
nit / up to you, but would it be clearer to mirror the cohere.Client param names and call  this base_url?

Reply:
good call - updated